packages/react-components/react-progress/src/stories/Progress/ProgressTypes.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { Progress } from '@fluentui/react-progress';
+
+const intervalDelay = 100;
+const intervalIncrement = 0.01;
+
+const useStyles = makeStyles({
+  container: {
+    ...shorthands.padding('20px', '0px'),
+  },
+});
+
+export const Types = () => {
```

Comment:
This could be called `Indeterminate` and just show how that mode works. The determinate case is already demonstrated in the default story.